### PR TITLE
use UTC for data

### PR DESCRIPTION
**Pull request by benghaziboy**
I hate everything

### CODE REVIEW

File: bulbs/special_coverage/management/commands/migrate_active_to_published.py [hunk 1]
```diff
@@ -10,14 +9,14 @@ class Command(BaseCommand):
 
     def get_month_start_date(self):
         """Returns the first day of the current month"""
-        now = get_central_now()
+        now = timezone.now()
```
Comment (mparent61) [inline bulbs/special_coverage/management/commands/migrate_active_to_published.py]: <3

**Comment by mparent61** [comment]
:+1:

**Comment by benghaziboy** [comment]
shut up mike

**Comment by MichaelButkovic** [comment]
LGTM

**Comment by mparent61** [comment]
which mike. Please be specific

**Comment by benghaziboy** [comment]
the dumb one

**Pull request merged**